devel/setup-e2e-deps.sh

```diff
@@ -54,6 +54,9 @@ echo "Installing ingress-nginx into the cluster..."
 echo "Loading vault into the cluster..."
 "${SCRIPT_ROOT}/addon/vault/install.sh" &
 
+echo "Installing sample-external-issuer into the cluster..."
+"${SCRIPT_ROOT}/addon/sample-external-issuer/install.sh" &
+
```

Comment:
Perhaps in another PR, but would it be possible to make this pluggable, so that I can supply my own external issuer which is run against the conformance suite?

Reply:
See https://github.com/jetstack/cert-manager/issues/3602